### ISSUE TITLE
cleaning up

### ISSUE THREAD

**Issue by ZebNemeth** [ZebNemeth/ComplexContagion#2]
Do we wish to make this repository more presentable and easier to navigate/understand for new visitors? I do, and will start at this when I feel I have time.

**Comment by AartvB** [comment]
Yes, you can do that if you want. Although I doubt if there will be any new visitors.
